test/Test/Time/Units.hs

```diff
@@ -7,14 +7,17 @@ module Test.Time.Units
        ( unitsTestTree
        ) where
 
-import Control.Exception (evaluate)
-import GHC.Real (Ratio ((:%)))
-import Test.Tasty (TestTree)
-import Test.Tasty.Hspec (Spec, anyException, describe, it, shouldBe, shouldThrow, testSpec)
+import           Control.Exception (evaluate)
```

Comment:
I pushed our company's `.stylish-haskell.yaml` to `master` branch. Could you please use this `.stylish-haskell.yaml` config for formatting  imports? All you need is just rebase on `master` in your fork/branch and then run `stylish-haskell` on this file again.